src/include/apparmor/aa-config.rb

```diff
@@ -33,408 +33,35 @@ def initialize_apparmor_aa_config(include_target)
       Yast.import "Label"
       Yast.import "Popup"
       Yast.import "Wizard"
+      Yast.import "Service"
     end
 
     def changeAppArmorState(aaEnabled)
-      error = nil
-      aaAction = ""
-
+      success = true
       if aaEnabled == true
-        aaAction = "apparmor:enable"
+        success = Yast::Service.start("apparmor")
+        if success
+          success = Yast::Service.enable("apparmor")
+        end
```

Comment:
you can write it as `success = Yast::Service.start("apparmor") && Yast::Service.enable("apparmor")`